compiler/extccomp.nim

```diff
@@ -645,7 +645,7 @@ proc compileCFile(conf: ConfigRef; list: CFileList, script: var Rope, cmds: var
     if optCompileOnly notin conf.globalOptions:
       add(cmds, compileCmd)
       let (_, name, _) = splitFile(it.cname)
-      add(prettyCmds, "CC: " & name)
+      add(prettyCmds, if hintCC in conf.notes: "CC: " & name else: nil)
```

Comment:
Strings are not ``nil``. This will soon stop to compile.

Reply:
And before you change that to

```nim

add(prettyCmds, if hintCC in conf.notes: "CC: " & name else: "")

```

Use instead:

```nim

if hintCC in conf.notes: 
  add(prettyCmds, "CC: " & name)

```

Reply:
replaced s/nil/"" but didn't replace to:
```
if hintCC in conf.notes: 
  add(prettyCmds, "CC: " & name)
```
since that'd lead to error in subsequent code in `let cmd = prettyCmds[idx]`.

Reply:
Ah ok. Terrible. :-)